modules/class-simplechart-plugin-versions.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This class manages when we show alerts to upgrade
+ */
+
+class Simplechart_Plugin_Versions {
+
+	private $_simplechart_releases_url = 'https://api.github.com/repos/alleyinteractive/wordpress-simplechart/releases';
+
+	public function __construct() {
+		add_action( 'admin_init', array( $this, 'simplechart_check_for_new_version' ) );
+		add_action( 'wp_ajax_simplechart_dismiss_nag', array( $this, 'simplechart_dismiss_nag' ) );
+	}
+
+	public function simplechart_check_for_new_version() {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+
+		if ( false === ( $value = get_transient( 'simplechart_plugin_version_remote' ) ) ) {
```

Comment:
Even though this variable isn't being used inside the block - we could give this a new name e.g. `$remove_version` to improve on readability